swingset/src/main/java/com/nqadmin/swingset/models/AbstractComboBoxListSwingModel.java

```diff
@@ -960,23 +944,28 @@ private boolean remove(Object _listItem) {
 	 * @return the previous contents of the list item at the specified position
 	 */
 	// NOTE: setElem(listItem, elemIndex, newElem)
-	// is a problem because need listitemindex for fireContentsChanged
+	// is a problem because this method wants to modify something inside
+	// of a listItem, so the listItem's identity, via '==',
+	// doesn't change. Best way to insure that changes are
+	// correctly detected is to make a listItem immutable.
+	// So make a copy/clone of the listItem.
+	//
+	// I think this behavior can be conditional on whether or not
+	// glazed is used or if this is actually installed as a model.
+	// In the "right" places could do "modifyListItemWithSet = !installed".
+	// or "useClone = modifyListItemWithSet || !installed"
```

Comment:
Eliminate this reference to useClone in JavaDoc since you did away with the variable?